### ISSUE TITLE
Reproduce results for CGConv

### ISSUE THREAD

**Issue by Chen-Cai-OSU** [pyg-team/pytorch_geometric#816]
Hello, 

Another related question about CGConv. I looked at the original github repo for CGConv and compared it with your implementation. Your implementation is consistent with your documentation (all the math are right and same with the original paper) but different from code [here](https://github.com/txie-93/cgcnn/blob/master/cgcnn/model.py). For example, the batch normalization is used in forward of the ConvLayer class but not in your CGConv class. 

I was wondering when you implement a method, will you check with the original implementation, or do you mainly stick with the math in the paper? Sometimes some design choices may not be mentioned in the paper and this may cause some difference.

Hopefully, my question makes sense to you. I would like to use pytorch-geometric but also want to make sure I can replicate the result in the original paper.

**Comment by rusty1s** [comment]
Generally, I check for the original implementation, but must have missed the batch normalization here. Feel free to submit a PR to fix this if you are interested :)
